_includes/nav-button.html

```diff
@@ -0,0 +1 @@
+<a href="#nav">Contents &darr;</a>
```

Comment:
Can we set the symbol with `:after` so that it can be a Sass variable, along with `$nav-bar-children-prompt` and `$nav-bar-children-prompt-hide`? My thinking is that the choice of symbol is a look-and-feel thing, so might as well set it in the Sass.

I see you removed '$nav-bar-prompt' as a variable, which is where I was setting the word 'Contents' as the default value, and inserting it also with CSS. This was because some sites call for a word like 'Contents' (where the site is very booky) and some something like 'Menu' or even a symbol (like your unfavourite hamburger).  Any particular reason you don't want to set the word/prompt in this way?

I am also generally trying to avoid hard-coding in English where possible. Though real translations will take more thought than this.